### ISSUE TITLE
It would be better if you can provide info about how you divide your dataset.

### ISSUE THREAD

**Issue by ZhengHe-MD** [lemondan/HumanParsing-Dataset#7]
briefly, in order to compare different methods, we need a predetermined division of this dataset, though it may not make much difference. At least you should provide the ratio (ratios of ATR dataset and Fashionista dataset are provided in your paper. However, I didn't find exactly 7700 images in the dataset you provided, because there are only 7018 images if you substract 685 fashionista iamges, and 10003 Chictopia images from your total images 17706) of how you divide Chictopia10k into train, val and test set~ It would be great if you can just provide exactly the train, val and test set of your dataset. thank you!~

**Comment by ZhengHe-MD** [comment]
or what you did is 
using 1000 of the ATR dataset and 229 of the Fashionista dataset together as test set, 
700 of the ATR dataset as validation set,
and all the rest images, including Chictopia10k as train set?

**Comment by ZhengHe-MD** [comment]
never mind. I've found that in your ATR paper. close this.
